packages/botonic-core/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "../../node_modules/.bin/jest --coverage",
     "prepare": "node ../../preinstall.js",
     "lint": "npm run lint_ci -- --fix",
-    "lint_ci": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.js*'",
+    "lint_ci": "../../node_modules/.bin/eslint_d --cache --quiet '.*.js' '*.js' 'src/**/*.js*'",
```

Comment:
Is this duplication of the kind of files intended? Probably we can remove one `'*.js'`, right?

Reply:
nope, try "ls *.js" from bash and you'll see that the hidden files do not appear.
globs in package.json work likewise